civictechprojects/views.py

```diff
@@ -42,25 +50,26 @@ def to_tag_map(tags):
     return list(tag_map)
 
 
-def project_signup(request):
+def project_create(request):
     if not request.user.is_authenticated():
         return redirect('/signup')
-    if request.method == 'POST':
-        ProjectCreationForm.create_project(request)
-        return redirect('/index/?section=MyProjects')
-    else:
-        form = ProjectCreationForm()
 
-    template = loader.get_template('project_signup.html')
-    issues = get_tags_by_category('Issue(s) Addressed')
-    tag_map = to_tag_map(issues)
-    context = {
-        'form': form,
-        'issues': json.dumps(tag_map)
-    }
-    return HttpResponse(template.render(context, request))
+    ProjectCreationForm.create_project(request)
+    return redirect('/index/?section=MyProjects')
+
+
+def project_edit(request, project_id):
+    if not request.user.is_authenticated():
+        return redirect('/signup')
+
+    try:
+        ProjectCreationForm.edit_project(request, project_id)
+    except PermissionDenied:
+        return HttpResponseForbidden()
+    return redirect('/index/?section=AboutProject&id=' + project_id)
 
 
+# TODO: Remove when React implementation complete
```

Comment:
create a tracking task for this and reference it here?

Reply:
https://trello.com/c/gyaguFWh